src/main/java/uk/gov/hmcts/reform/em/annotation/domain/Bookmark.java

```diff
@@ -0,0 +1,102 @@
+package uk.gov.hmcts.reform.em.annotation.domain;
+
+import org.springframework.data.annotation.CreatedBy;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.Size;
+import java.io.Serializable;
+import java.util.UUID;
+
+/**
+ * A Bookmark.
+ */
+@Entity
+@Table(name = "bookmark")
+public class Bookmark implements Serializable {
+
+    @Id
+    private UUID id;
+
+    @Size(min = 1, max = 30)
+    @Column(name = "name", length = 30, nullable = false)
+    private String name;
+
+    @Column(name = "document_id", nullable = false)
+    private UUID documentId;
+
+    @CreatedBy
+    @Column(name = "created_by", nullable = false, length = 50, updatable = false)
+    private String createdBy;
+
+    @Column(name = "num", nullable = false)
+    // pdfjs specific location information
+    private int num;
+
+    @Column(name = "x_coordinate", nullable = false)
+    private double xCoordinate;
+
+    @Column(name = "y_coordinate", nullable = false)
+    private double yCoordinate;
```

Comment:
I think the x and y coordinates might be null sometimes

Reply:
ok, will allow it to be nullable